data/og/window_Firefox_48.0_OSX_10.6.json

```diff
@@ -0,0 +1,2 @@
+{
+}
```

Comment:
Did this data collection fail? The database is empty. If we can't collect data from this version yet, let's not check in an empty database (and let's also file a bug).